### DISCUSSION TITLE
What font does vercel using in there dashboard

### DISCUSSION THREAD

**Discussion by 2820402** [vercel/community#1255] [Help] [answered]
I want to know what font does vercel using in there dashboard I think it's Open Sans but don't know exactly.

**Comment by amyegan** [comment, marked as answer]
Hi @2820402! The font is [Inter](https://fonts.google.com/specimen/Inter). If you're using Next.js, you can load an optimized version of the [Google Font](https://nextjs.org/docs/basic-features/font-optimization#google-fonts) with [`@next/font`](https://nextjs.org/docs/api-reference/next/font)
